README.md

```diff
@@ -44,16 +44,19 @@ For more details, see [building from source](https://github.com/Xpra-org/xpra/tr
 ---
 
 # Usage
+## Initial requirements
+`xpra` must be installed on the client and the host (version matters).
```

Comment:
That's not quite true, you can use the html5 client in which case only the server is required...

Reply:
Try this then:

`xpra` must be installed on the client and the host (version matters).  

You can use the html5 client in which case xpra is only required on the host.  People that are using the html5 client would already be somewhat experienced xpra users.

Reply:
I've done another commit.  Just using the github interface which is a bit shit, was only supposed to be a quick adjustment.